### ISSUE TITLE
500 Error when searching and not selecting a result from the popup

### ISSUE THREAD

**Issue by rnegron** [jpadilla/notaso#108]
**Error**: **'Manager' object has no attribute 'search'**

Line 24 in notaso/search/views.py, _get_queryset_

`qs = Professor.objects.search(term, raw=True)`

```
Request Method: GET
Request URL: http://localhost:8000/search/?q=erwrq

Django Version: 1.11.15
Python Version: 2.7.15
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.sites',
 'debug_toolbar',
 'django_extensions',
 'storages',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'allauth.socialaccount.providers.facebook',
 'allauth.socialaccount.providers.twitter',
 'djangosecure',
 'import_export',
 'raven.contrib.django.raven_compat',
 'rest_framework',
 'rest_framework_swagger',
 'corsheaders',
 'notaso.home',
 'notaso.users',
 'notaso.universities',
 'notaso.professors',
 'notaso.departments',
 'notaso.comments',
 'notaso.search')
Installed Middleware:
(u'raven.contrib.django.middleware.DjangoRestFrameworkCompatMiddleware',
 u'raven.contrib.django.middleware.SentryMiddleware',
 'djangosecure.middleware.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware',
 'corsheaders.middleware.CorsMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware')



Traceback:

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py" in _legacy_get_response
  249.             response = self._get_response(request)

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python2.7/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/django/views/generic/list.py" in get
  160.         self.object_list = self.get_queryset()

File "/app/notaso/search/views.py" in get_queryset
  24.                 qs = Professor.objects.search(term, raw=True)

Exception Type: AttributeError at /search/
Exception Value: 'Manager' object has no attribute 'search'
```

**Comment by jpadilla** [comment]
Should be fixed in #107

**Comment by rnegron** [comment]
Was indeed fixed in #107
